### PR TITLE
Add Result Submission

### PR DESCRIPTION
**Pull request by jparksecurity**
@lkasvr You can see the ID of the tags on the `ResultSubmission.tsx` to work on `puppeteer`.

I'll add more logic and some design before marking it as ready.

### CODE REVIEW

**Comment by lkasvr** [comment]
We can use anothers css or query selectors depending on our use case, the puppeteer will be fit anyway. 

But, I think that you choice for the ID is better for now.

**Comment by jparksecurity** [comment]
> We can use anothers css or query selectors depending on our use case, the puppeteer will be fit anyway.
> 
> But, I think that you choice for the ID is better for now.

I see. I didn't consider what would be the best way. Just use what works best for you.

File: database.json
Comment (jparksecurity) [inline database.json:1]: I add this to demonstrate the script is able to submit the form

**Comment by jparksecurity** [comment]
Also, I fixed the typo in the folder name.

**Comment by lkasvr** [comment]
> We can use anothers css or query selectors depending on our use case, the puppeteer will be fit anyway.
> 
> But, I think that you choice for the ID is better for now.

Maybe we can use the css selector o another query selector just based on the legacy HTML. I mean whatever the legacy html is using (legacy html = HTML of the real system)

**Review by lkasvr — APPROVED** [review]

**Pull request merged**